src/main/resources/db/migration/V0.3__norms_create_tables.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE
+  norms (guid uuid PRIMARY KEY, long_title VARCHAR(255));
+
+CREATE TABLE
+  articles (
+    guid uuid PRIMARY KEY,
+    title VARCHAR(255),
+    marker VARCHAR(20),
```

Comment:
I thought yesterday you said we should be more open about the marker until we understand it better. 😛 
But I guess it is better to start with a small variable character sequence and convert it into a larger one than the other way around. 😆

Reply:
I am still open about it as a text, but discussed that with Victor and saw that for the mean time a varchar of 20 is actually a good start. We can increase that later as you said. In the database that shouldn't be a problem as long as we are not changing the type.